lib/codecs/tests/avro.rs

```diff
@@ -18,9 +18,7 @@ use vector_core::{config::LogNamespace, event::Event};
 #[case(true)]
 #[case(false)]
 fn roundtrip_avro_fixtures(
-    #[files("tests/data/avro/generated/*.avro")]
-    #[exclude(".*(date|fixed|time_millis|timestamp_micros|timestamp_millis).avro")]
```

Comment:
Let's add back the `exclude` since it makes it obvious which cases are unsupported.